Project-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Header.css';
+
+
+function ImgThumbnail() {
+    return (
+        <img src="https://tk-assets.lambdaschool.com/1c1b7262-cf23-4a9f-90b6-da0d3c74a5c6_lambdacrest.png"></img>
```

Comment:
```suggestion
        <img src="https://tk-assets.lambdaschool.com/1c1b7262-cf23-4a9f-90b6-da0d3c74a5c6_lambdacrest.png"/>
```
You can self close it to make it just that much better